indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -636,17 +644,31 @@ def _process(self):
         return jsonify_data(**self.list_generator.render_list())
 
 
-class RHRegistrationsModifyStatus(RHRegistrationsActionBase):
-    """Accept/Reject selected registrations."""
+class RHRegistrationsApprove(RHRegistrationsActionBase):
+    """Accept selected registrations from registration list."""
 
     def _process(self):
-        approve = request.form['flag'] == '1'
         for registration in self.registrations:
-            _modify_registration_status(registration, approve)
-        flash(_("The status of the selected registrations was updated successfully."), 'success')
+            _modify_registration_status(registration, approve=True)
+        flash(_("The selected registrations was successfully '{}'.").format('approved'), 'success')
```

Comment:
Feel free to make the string more clear ("The selected registrations were successfully approved" or something like that), but do not break i18n like this particular change does.